server/repository/src/db_diesel/item_direction_row.rs

```diff
@@ -0,0 +1,105 @@
+use super::item_direction_row::item_direction::dsl::*;
+use super::item_link;
+use crate::Delete;
+use crate::RepositoryError;
+use crate::StorageConnection;
+use crate::Upsert;
+use diesel::prelude::*;
+use serde::{Deserialize, Serialize};
+
+table! {
+    item_direction (id) {
+        id -> Text,
+        item_link_id -> Text,
+        directions -> Text,
+        priority -> BigInt,
+    }
+}
+
+#[derive(
+    Clone, Insertable, Queryable, Debug, PartialEq, AsChangeset, Eq, Serialize, Deserialize,
+)]
+#[diesel(table_name = item_direction)]
+#[diesel(treat_none_as_null = true)]
```

Comment:
This line is a little bit pointless with no nullable columns :)

Reply:
Cargo cult 😆